### PR TITLE
Upgrade to 3.0 & remove ember-cli fork dependency

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
This PR gives us,
- A shiny new ember 3.0 upgrade 🦄 
- A in-repo-addon that uses exisiting ember-cli features to address part of #65. (The PR to the prember is still required)

### CODE REVIEW

File: lib/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (rwjblue) [inline lib/.eslintrc.js]: Probably shouldn’t need this file. Should be able to modify the top level eslint config’s overrides to do what you need here (basically adding `lib/*/index.js` to the “node files” entry)...

File: lib/content-docs-generator/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/* eslint-env node */
```
Comment (rwjblue) [inline lib/content-docs-generator/index.js]: This shouldn’t be needed...

**Comment by sivakumar-kailasam** [comment]
@rwjblue both of those were generated by the default blueprint. Will raise a PR to the cli repo.

File: .travis.yml [hunk 3]
```diff
@@ -15,23 +15,24 @@ addons:
...
 env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
+  - npm install -g npm@5
+  - npm config set spin false
```
Comment (acorncom) [inline .travis.yml]: @mansona @sivakumar-kailasam and I discussed different aspects of this commit and our long-term approach to PRs, reviews and `yarn`/`npm` offline. We'll be working out ways for us to all stay on the same page as a team, but fundamentally miscommunications here were unintentional.

**Comment by serenaf** [comment]
@acorncom Hey David, our blocker (the nav-bar) has been resolved 🎉  @sivakumar-kailasam  Do you mind rebasing this branch and then we can merge it?
Also ❤️ on communication re npm and yarn. They are both great :)

**Comment by sivakumar-kailasam** [comment]
@serenaf 👍  gtg

**Pull request merged**